### ISSUE TITLE
Fine tune inception without TFRecords.

### ISSUE THREAD

**Issue by chrisrn** [tensorflow/tensorflow#5519]
I wonder if I can fine-tune the inception model without using TFRecords format for my dataset in the inception_train script. I have a placeholder for images and a placeholder for labels and during training I feed them in sess.run(). I get low precision on a binary classification problem so I wonder if this approach is wrong. In stackoverflow noone answered. Thanks in advance!

**Comment by sherrym** [comment]
Hi @chrisrn ,

You should be able to as long as you provide the images in the same format and distribution as expected by the pre-trained model. By "as expected" I meant if the values used for training were in the range of 0-255, your dataset will need to be in the same range.

Sherry

**Comment by chrisrn** [comment]
I feed the images with size 299x299x3 and values in range [-1, 1] like the image_processing script. Am I right? I need to confirm that I don't feed them wrong in order to proceed in next steps of training.

**Comment by girving** [comment]
Questions like this do belong on Stackoverflow, since this isn't a bug report or feature request.
